### ISSUE TITLE
[Web] Certain content in the discover section opens within Apollos instead of newspring.cc

### ISSUE THREAD

**Issue by IsaacHardy** [NewSpring/Apollos#607]
## Acceptance Criteria

- [ ] All content in Apollos web should open the specific piece of content at Newspring.cc

## Context

Some of the content you find in the discover feature within Apollos is opened in apollos instead of newspring.cc. I have not been able to determine any common denominators with which content opens in app or newspring.cc

**Comment by richarddubay** [comment]
@IsaacHardy I wonder if this has to do with how the Discover section works in the app. Content like articles and devos that we actually have content for in the app open in the app but things like locations open in an in-app browser. I bet it's a similar thing.

**Comment by conrad-vanl** [comment]
Yeah, we should likely add a `Redirect` method within our actual content views or something. you can go straight to content like this if you have a valid url.
